tools/browser_compat_data_update.py

```diff
@@ -0,0 +1,113 @@
+import json
+from sys import maxsize
+import requests
+from pathlib import Path
+
+
+def parse_version(version, browser):
+    version = [int(x) for x in version.split(".")]
+    if browser == "safari" or browser == "nodejs":
+        return sum(x * 100 ** (2 - i) for i, x in enumerate(version))
+    else:
+        return version[0]
+
+
+interesting_browsers = ["chrome", "firefox", "safari", "nodejs"]
```

Comment:
We tend use single quote string in our python scripts.

Reply:
Yeah I had them but then file got ugly so I ran Black autoformatter and it changed them to double quotes.

Do we have preferred autoformatter & config for Python files?

Reply:
I've used `yapf` in the past and there is the `.style.yapf` in the top level.

Reply:
Thanks. I'll mark as unresolved and use it to reformat a bit later.